disnake/client.py

```diff
@@ -760,6 +768,159 @@ def dispatch(self, event: str, *args: Any, **kwargs: Any) -> None:
         else:
             self._schedule_event(coro, method, *args, **kwargs)
 
+        for event_ in self.extra_events.get(method, []):
+            self._schedule_event(event_, method, *args, **kwargs)
+
+    def add_listener(self, func: CoroFunc, name: Union[str, Event] = MISSING) -> None:
+        """The non decorator alternative to :meth:`.listen`.
+
+        .. versionchanged:: 2.10
+            The definition of this method was moved from :class:`.ext.commands.Bot`
+            to the :class:`.Client` class.
```

Comment:
could you also make the other methods' docstrings consistent with the `versionchanged` in this one?

Reply:
https://github.com/DisnakeDev/disnake/pull/1066/commits/2f4e3573e33c9661119d5d815e859ab15b9f4485